core/genesis.go

```diff
@@ -244,7 +244,7 @@ func (g *Genesis) configOrDefault(ghash common.Hash) *params.ChainConfig {
 	case ghash == params.GoerliGenesisHash:
 		return params.GoerliChainConfig
 	case ghash == params.YoloV1GenesisHash:
```

Comment:
Nit, pls rename to YoloV2. I know it didn't change, but seems nicer to get rid of all v1 mentions in the code.

Reply:
Yeah... I mean, we're going to have to put a new hash in there, at some point. I can rename it meanwhile -- or just comment it out?